src/pages/waiting-room-to-car/components/eyesight-test/eyesight-test.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Input, Output, EventEmitter, OnChanges } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'eyesight-test',
+  templateUrl: 'eyesight-test.html',
+})
+export class EyesightTestComponent implements OnChanges {
+
+  @Input()
+  testState: string;
+
+  @Input()
+  formGroup: FormGroup;
+
+  @Output()
+  eyesightTestResultChange = new EventEmitter<string>();
+
+  formControl: FormControl;
+
+  ngOnChanges(): void {
+    if (!this.formControl) {
+      this.formControl  = new FormControl('', [Validators.required]);
+      this.formGroup.addControl('eyesightCtrl', this.formControl);
+    }
+    this.formControl.patchValue(this.testState);
+  }
+  eyesightTestResultChanged(result: string): void {
+    if (this.formControl.valid) {
+      this.eyesightTestResultChange.emit(result);
+    }
+  }
+
+  get invalid(): boolean {
```

Comment:
I was following a pattern established in the office screen - rather than have two pages with inconsistent approaches (or an even bigger PR :) ) I chose to go with the same pattern. 
It's also used to display the error formatting so would always be used as !valid so invalid seems reasonable.

Reply:
It's really just preference, however IMO building a function which returns _valid_ result is always easier and more simple to reason about. And you can use the exclamation mark to negate it where ever you need to.